hikey_debian.mk

```diff
@@ -226,7 +226,7 @@ PERF_FLAGS := CROSS_COMPILE_HOST=$(CROSS_COMPILE_NS_USER) \
 	TA_DEV_KIT_DIR=$(OPTEE_OS_TA_DEV_KIT_DIR)
 
 aes-perf: optee-os optee-client
-	$(MAKE) -C -j$(NPROC) $(AESPERF_PATH) $(PERF_FLAGS)
+	$(MAKE) -j$(NPROC) -C $(AESPERF_PATH) $(PERF_FLAGS)
```

Comment:
In fact I found it very convenient to be able to build it alone for development and test.